test/fixtures/responses/get-trustlines-more-than-400-items.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Nit, this json fixture is only one item. Could this be renamed to something better aligned, (just brainstorming here) `trustline-response.json` or `trustline-item.json`?